modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/InterceptingExecutionMediator.java

```diff
@@ -39,8 +45,8 @@ public InterceptingExecutionMediator(ExecutionMediator intercepted) {
   }
 
   @Override
-  public Object execute(OperationExecutor executor, ExecutionContextAdapter context) throws Throwable {
-    Object resultValue = intercepted.execute(executor, context);
+  public Publisher<Object> execute(OperationExecutor executor, ExecutionContextAdapter context) {
+    Object resultValue = from(intercepted.execute(executor, context)).block();
```

Comment:
What happens with exceptions that might be thrown here by block?  They should be caught and return error Mono shouldn't they? (same in other use like this)